aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -298,6 +298,59 @@ Tensor& sigmoid_out(Tensor& result, const Tensor& self) { return unary_op_impl_o
 Tensor sigmoid(const Tensor& self) { return unary_op_impl(self, at::sigmoid_out);  }
 Tensor& sigmoid_(Tensor& self) { return unary_op_impl_(self, at::sigmoid_out);  }
 
+Tensor& heaviside_tensor_out(Tensor& result, const Tensor& self, const Tensor& val) {
```

Comment:
Heaviside is a binary, not unary, op, so it goes in BinaryOps.cpp.

Reply:
Addressed, Moved to BinaryOps.cpp.